plugins/wasi_nn/wasinnenv.h

```diff
@@ -145,13 +146,51 @@ struct WasiNNEnvironment :
     FOR_EACH_BACKEND(EACH)
 #undef EACH
         std::monostate {
-  WasiNNEnvironment() noexcept {
-    NNGraph.reserve(16U);
```

Comment:
Why does this section remove?

Reply:
I put the constructor definition into .cpp file. I thought they meant the same thing.
https://github.com/CaptainVincent/WasmEdge/blob/0b528de8307b3032acc3796b45a45937d12f638f/plugins/wasi_nn/wasinnenv.cpp#L77